src/main/java/uk/gov/hmcts/darts/audit/api/AuditActivity.java

```diff
@@ -33,7 +33,9 @@ public enum AuditActivity {
     AMEND_TRANSCRIPTION_WORKFLOW(24),
     CREATE_RETENTION_POLICY(25),
     EDIT_RETENTION_POLICY(26),
-    REVISE_RETENTION_POLICY(27);
+    REVISE_RETENTION_POLICY(27),
+    RUN_JOB_MANUALLY(28),
+    ENABLE_DISABLE_JOB(29);
```

Comment:
Is it ok we don't differentiate between enabling and disabling tasks?

Reply:
yeah I thought the same but the requirements were not clear I will check

Reply:
@jonathan-littlewood is back tomorrow

Reply:
@jackmaloney see @jonathan-littlewood response:-

'I don't think it is necessary to have separate "enable" and "disable" events since the new state will be recorded on the Envers table. I've added a comment along those lines to the ticket'